### ISSUE TITLE
hotkey no longer working after upgrade and debugger message

### ISSUE THREAD

**Issue by rabihmb** [flutter/flutter#49552]
I'm using the Android Studio, I can confirm that when upgraded the flutter to the latest and got the issues below:

1. Hotkey not working
2. this app is only meant to be run under flutter debugger
    
## Steps to Reproduce

open any project and run it

**Expected results:** 

the hotkey working as usual and don't appear the message!

**Actual results:**
1. Hotkey not working
2. getting a message "this app is only meant to be run under flutter debugger"

<summary>Logs</summary>

```
Launching lib\main.dart on SM N920C in debug mode...
Running Gradle task 'assembleDebug'...
√ Built build\app\outputs\apk\debug\app-debug.apk.
Syncing files to device SM N920C...
Syncing files to device SM N920C...
I/flutter (23460): Load en
D/ViewRootImpl@2be2fb2[MainActivity](23460): Relayout returned: oldFrame=[0,0][1080,1920] newFrame=[0,0][1080,1920] result=0x1 surface={isValid=true 491992236544} surfaceGenerationChanged=false
```

```
Analyzing asisc...

   info - This class (or a class that this class inherits from) is marked as '@immutable', but one or more of its instance fields aren't final: MyApp.myTranslations - lib\main.dart:27:7 - must_be_immutable

1 issue found. (ran in 3.3s)
```

```

C:\Users\user1>flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel master, v1.14.5-pre.39, on Microsoft Windows [Version 10.0.18363.592], locale en-US)
[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[√] Android Studio (version 3.5)
[√] VS Code (version 1.41.1)
[√] Connected device (1 available)
• No issues found!
```

**Comment by jonahwilliams** [comment]
You haven't actually included any run logs. nevertheless, I'm turning off fast-start by default so part of this might be fixed by https://github.com/flutter/flutter/pull/49554

**Comment by rabihmb** [comment]
> 
> 
> You haven't actually included any run logs. nevertheless, I'm turning off fast-start by default so part of this might be fixed by #49554

Expand the Log you found the rest!

In that case, should I enable the flag back to true? 

clarify please!

**Comment by iapicca** [comment]
Hi @rabihmb 

if you believe the issue is caused by Android Studio plugin please
follow the indication of [jetbrains' Flutter plugin](https://plugins.jetbrains.com/plugin/9212-flutter) 
opening an issue on the dedicated [github](https://github.com/flutter/flutter-intellij/issues);

if instead you think the problem might be with flutter itself
please provide your `flutter doctor -v`
and your `flutter run --verbose` 
Thank you

**Comment by jonahwilliams** [comment]
Fixed via #49554
